doc/whats-new.rst

```diff
@@ -3666,7 +3666,7 @@ Enhancements
   ``fillna`` works on both ``Dataset`` and ``DataArray`` objects, and uses
   index based alignment and broadcasting like standard binary operations. It
   also can be applied by group, as illustrated in
-  :doc:`Fill missing values with climatology <examples/weather-data>`.
+  :ref:`/examples/weather-data.ipynb#Fill-missing-values-with-climatology`.
```

Comment:
Thanks! I guess I was missing the first `/`

Reply:
yeah, `nbsphinx` seems to require it being an "absolute" path with the root being the directory `conf.py` is in.